superset-frontend/spec/javascripts/addSlice/AddSliceContainer_spec.tsx

```diff
@@ -17,14 +17,16 @@
  * under the License.
  */
 import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
```

Comment:
You can `import { styledMount as mount } from 'spec/helpers/theming';` as a shortcut, so that you don't have to import the ProviderWrapper or supersetTheme in this file.

Reply:
... and now I see you did that elsewhere. Maybe this one's different for good reason. ¯\\\_(ツ)_/¯

Reply:
Nice catch! I'll use `styledMount` here also.